### PR TITLE
Remove OpenRA.PostProcess.exe.

### PR DESCRIPTION
**Pull request by pchote**
I introduced OpenRA.PostProcess.exe in #16451 with the goal of using it + Mono.Cecil to work around the missing reflection APIs in netstandard2. This is no longer required now that we are planning to move directly to netstandard2.1, and the recently announced [Source Generators](https://devblogs.microsoft.com/dotnet/introducing-c-source-generators/) feature gives us a simpler route in the long term to get the other benefits.

As it currently stands, PostProcess's only job is to set the `/LARGEADDRESSAWARE` flag, which is only used on 32bit windows, and is only needed on the Game.exe. We no longer actively support using the 32bit build while developing (only when packaging), so we can simplify the toolchain by moving this to the windows packaging. This makes life a bit simpler for #17989.

### CODE REVIEW

File: OpenRA.sln [hunk 1]
```diff
@@ -19,8 +19,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenRA.Mods.Common", "OpenR
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenRA.Test", "OpenRA.Test\OpenRA.Test.csproj", "{6CB8E1B7-6B36-4D93-8633-7C573E194AC4}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenRA.PostProcess", "OpenRA.PostProcess\OpenRA.PostProcess.csproj", "{EE63AF7E-92EA-48FB-81E2-53D7D92F8050}"
```
Comment (teinarss) [inline OpenRA.sln:22]: Please remove rows 60-63 with the guid `EE63AF7E-92EA-48FB-81E2-53D7D92F8050`

**Review by teinarss — CHANGES_REQUESTED** [review]

**Comment by pchote** [comment]
Fixed.

**Review by teinarss — APPROVED** [review]

**Review by abcdefg30 — APPROVED** [review]
Ok.

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/10846bb96c73fec2eb082579dcd05061a0cc1e39...f83f734068e77f1a043ea8d8ab079afca00dd8ed)

**Pull request merged**